src/specs/specs-other.json

```diff
@@ -1,8 +1,24 @@
 [
   "https://streams.spec.whatwg.org/",
+  "https://w3c.github.io/accname/",
+  "https://w3c.github.io/core-aam/",
   "https://w3c.github.io/editing/contentEditable.html",
+  "https://w3c.github.io/editing/execCommand.html",
```

Comment:
Not sure this should be added. @tidoust what do you want to do for specs that aren't implemented yet or that are abandoned? Add them to blacklist.json?

Reply:
I would put specs that aren't implemented yet in the list of specs crawled, and add specs that are abandoned to blacklist.json... unless they are implemented and on a long transition path to being abandoned perhaps.

In this particular case, I think I'd leave it out since we have planned replacements (contentEditable and input events) in the list. But if you feel that's useful, fine to have it in!

Reply:
I don't think `execCommand` is ever going to be removed, but nobody has any appetite for working on the spec, so I'll drop this. Might add it if later some reference to it is unresolved.